src/lib/lite_base/nat.ml

```diff
@@ -31,10 +43,27 @@ module Make (M : Input_intf) = struct
           go init t 0 ) }
 
   let fold t = Fold_lib.Fold.group3 ~default:false (fold t)
+
+  module Importable = struct
+    type t = Stable.Latest.t [@@deriving eq, sexp, compare]
+
+    let length_in_triples, fold = (length_in_triples, fold)
+  end
 end
 
-module Make32 () = Make (struct
-  type t = Core_kernel.Int32.t [@@deriving bin_io, eq, sexp, compare]
+module Input_32_V1 = struct
+  module Stable = struct
+    module V1 = struct
+      module T = struct
+        type t = Core_kernel.Int32.t
+        [@@deriving bin_io, eq, sexp, compare, version {asserted}]
+      end
+
+      include T
+    end
+  end
+
+  include Stable.V1
```

Comment:
I think this can be omitted, since you'll always pass a particular version to `Make`, rather than the whole module.